pkg/extensions/search/common/oci_layout.go

```diff
@@ -384,7 +384,7 @@ func (olu BaseOciLayoutUtils) GetExpandedRepoInfo(name string) (RepoInfo, error)
 
 		tag, ok := man.Annotations[ispec.AnnotationRefName]
 		if !ok {
-			tag = "latest"
+			continue
```

Comment:
add a log here where you specify that you are skipping manifest with digest X because it doesn't have a tag

Reply:
done.